lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -296,7 +296,6 @@ def create_thread(request, course_id, commentable_id):
 
     data = thread.to_dict()
 
-    add_courseware_context([data], course)
```

Comment:
It occurs a few lines up already, so having it here twice ends up being redundant.